### PR TITLE
[scnlib] Update to 1.0

### PR DESCRIPTION
**Pull request by walge-laine**
[scnlib] Update to 1.0

- #### What does your PR fix?  
  Update scnlib to version 1.0

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

File: ports/scnlib/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "scnlib",
-  "version-string": "0.4",
+  "version": "1.0",
   "port-version": 1,
```
Comment (Thomas1664) [inline ports/scnlib/vcpkg.json:4]: Please remove `port-version`. You also need to revert the changes via `vcpkg x-add-version` and run the command again.

**Review by Thomas1664 — CHANGES_REQUESTED** [review]

**Review by Thomas1664 — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]
